app/actions/navigation/index.js

```diff
@@ -181,3 +182,31 @@ export function goToModalAccountSettings() {
         }, getState);
     };
 }
+
+export function goToCreateChannel(channelType) {
+    return async (dispatch, getState) => {
+        closeDrawers()(dispatch, getState);
+        let type;
+        let route;
+        switch (channelType) {
+        case Constants.OPEN_CHANNEL:
+            type = NavigationTypes.NAVIGATION_PUSH;
```

Comment:
Elias explained this to me in a 1-1. I missed the fact that the channel drawer works differently on mobile (+ next to Public Channels opens the More Channels modal instead of having a separate button for both)